### PR TITLE
Various typos, updates to reflect Gent-specific situations.

### PR DESCRIPTION
**Pull request by epauwels**

### CODE REVIEW

**Comment by boegel** [comment]
@itkovian: leave this one to me

@epauwels: let us know when you're done taking into account the remarks from #107 (try not to expand this PR to more files)

**Comment by itkovian** [comment]
@boegel: OK :p

**Comment by itkovian** [comment]
@boegel: maybe take a peek again? :-p

File: intro-HPC/ch_fine_tuning_job_specifications.tex [hunk 1]
```diff
@@ -338,6 +352,9 @@ \subsection{Number of processors}
 processor : 7
 \end{prompt}
 
+\underbar{Remark}: Unless you want information of the login nodes, you'll have to issue these commands
+on one of the workernodes. This is most easily achieved in an interactive job, see the chapter on
+Running interactive jobs.
```
Comment (boegel) [inline intro-HPC/ch_fine_tuning_job_specifications.tex]: isn't there a `\ref{...}` you can use?

**Comment by boegel** [comment]
Did a full review again, looks excellent to me, so going in. 

Thanks for the reminder @itkovian, and of course @epauwels for the editing!

**Pull request merged**